html-syntax/index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<body>
```

Comment:
Hey Simon, make sure to include the rest of the HTML skeleton components here, such as the html and head elements :)

Reply:
<img width="951" alt="Screen Shot 2021-08-16 at 2 43 52 PM" src="https://user-images.githubusercontent.com/88165545/129633420-3e83a4d4-4617-4d7c-befd-987188ed7bc0.png">